fastai/utils/get_pynvml.py

```diff
@@ -0,0 +1,20 @@
+"""Get OS specific nvml wrapper. On OSX we use pynvx as drop in replacement for pynvml"""
+
+import platform
+from ..script import *
+
+def get_pynvml():
```

Comment:
probably best not to call the module and function by the same name, as both share the same symbol.

In ipyexperiments I called the module pynvml_gate.py, but I'm not attached to it.
https://github.com/stas00/ipyexperiments/blob/master/ipyexperiments/utils/pynvml_gate.py

Reply:
This looks like a useful terminology. I'll adapt to that.